assets/helpers/tracking/conversions.js

```diff
@@ -9,8 +9,10 @@ export default function trackConversion(
   participations: Participations,
   currentRoute: string,
 ) {
-  // Fire GTM conversion events
-  successfulConversion(participations);
+  if (navigator.doNotTrack !== '1') {
```

Comment:
Not really related to your code but it's worrying to me that (as you mentioned in person) you had to add this code in because an error in `successfulConversion` was stopping execution of further code

@Ap0c @rupertbates this makes me think we're missing `try/catch` around the contents of this function? unless I'm missing something?